src/boot/efi/util.c

```diff
@@ -769,6 +770,126 @@ EFI_STATUS make_file_device_path(EFI_HANDLE device, const char16_t *file, EFI_DE
         return EFI_SUCCESS;
 }
 
+static const EFI_GUID QEMU_KERNEL_LOADER_FS_MEDIA_GUID = {
```

Comment:
Just makes these defines:
```
#define QEMU_BOOT_ORDER_GUID { ... }
```

And use them like this:
```
 &(EFI_GUID) QEMU_BOOT_ORDER_GUID
```

Reply:
Wondering why?  The uuids end up being compiled into the binary either way ...

Reply:
You're creating global variables when there is no need. C99 compound literals for these are so much nicer.